app/data/config/mainnet-go-opera/start-node.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+# download genesis file
+wget https://download.fantom.network/mainnet-109331-no-history.g
+
+./opera --genesis=mainnet-109331-no-history.g --db.preset ldb-1 --syncmode snap --http --http.addr="0.0.0.0" --http.corsdomain="*" --http.api=eth,web3,net,txpool,ftm
```

Comment:
API node isn't supposed to run snap:
https://docs.fantom.foundation/node/common-q-and-a#sync-modes